packages/pass-style/src/types.js

```diff
@@ -96,8 +96,9 @@ export {};
  */
 
 /**
+ * @template {string} [T=string] tag
  * @typedef {{
- *   [Symbol.toStringTag]: string,
+ *   [Symbol.toStringTag]: T,
  *   payload: Passable,
  *   [passStyle: symbol]: 'tagged' | string,
  * }} CopyTagged
```

Comment:
Won't this equate CopyTagged with CopyTagged\<string>, which is novel and unexpected because "string" is not a known tag?

Reply:
hmm, I don't think `CopyTagged<string>` would satisfy the template because the parameter wouldn't be a string itself. It would have to by `CopyTagged<'string'>`. That's not a known tag but it was valid before, right? (i.e. not a regression).